openpype/hosts/blender/api/ops.py

```diff
@@ -384,12 +470,21 @@ def draw(self, context):
         layout.operator(LaunchWorkFiles.bl_idname, text="Work Files...")
         # TODO (jasper): maybe add 'Reload Pipeline', 'Reset Frame Range' and
         #                'Reset Resolution'?
+        layout.operator(
+            WM_OT_DownloadLastWorkfile.bl_idname, text="Update Workfile"
+        )
 
 
 def draw_avalon_menu(self, context):
     """Draw the Avalon menu in the top bar."""
 
-    self.layout.menu(TOPBAR_MT_avalon.bl_idname)
+    self.layout.menu(
+        TOPBAR_MT_avalon.bl_idname,
+        icon="ERROR"
+        if bpy.context.window_manager.is_workfile_out_of_date
```

Comment:
`context` is passed as argument.
```suggestion
        if context.window_manager.is_workfile_out_of_date
```